.github/workflows/build.yaml

```diff
@@ -90,3 +99,23 @@ jobs:
       run: |
         cargo install --path .
         agora --version
+
+    - name: Package
+      id: package
+      if: startsWith(github.ref, 'refs/tags/')
+      shell: bash
+      env:
+        TARGET: ${{ matrix.target }}
+        REF: ${{ github.ref }}
+        OS: ${{ matrix.os }}
```

Comment:
I think it'd be nicer to pass these arguments in through command line flags to `./bin/package`. (Command-line parsing in `bash` is pretty bad, so maybe we should only consider that if we re-write `./bin/package` in a different language.)

Reply:
That's what I was thinking, command line parsing in bash is pretty bad, and using named environment variables with `set -u` is less error prone.

I think that `bin/package` should be rewritten in Rust. I've made multiple errors making changes to it. What do you think about leaving it in bash for now, and switching to rust some time after stir/cradle has its first relatively stable release?

Reply:
:) Yeah, I was thinking about using `stir` for this too... sounds good to me.